init.el

```diff
@@ -84,23 +100,24 @@
 ;; == whitespace ===============================================================
 (use-package whitespace
   :ensure t
-  :init
-    (setq whitespace-style '(face tabs empty trailing lines-tail))
-    (setq whitespace-line-column 100)
-    (global-whitespace-mode 1)
-    )
+  )
 
-;; == iedit ===========================================================================
-(use-package iedit
+
+(use-package clang-format
   :ensure t
+  :config
+    (setq clang-format-executable "/build/ltesdkroot/Tools/Tools/clang/CLANG_3.6_003/bin/clang-format")
```

Comment:
create string variable at the beginning of file contain this path and use it here instead of using raw text directly